Makefile

```diff
@@ -29,10 +29,11 @@ stop:
 
 # Run tests for the library
 test:
-	poetry export -f requirements.txt --without-hashes --with test --output requirements.txt
-	docker compose -f docker-compose.dev.yml up -d --build --wait
-	docker compose exec -T backend pytest --cov=app
-	docker compose -f docker-compose.dev.yml down
+	-poetry export -f requirements.txt --without-hashes --with test --output requirements.txt
+	-docker compose -f docker-compose.dev.yml up -d --build --wait
+	-sleep 30
```

Comment:
Isn't the whole point of the --wait and the healthcheck that we don't need to sleep? :smile:

Reply:
I still have many issues with the database initialization by alembic  if I don't add the sleep :/

Reply:
it's not consistent, it happens one time out of two

Reply:
really? I think we need to investigate that in another PR, sleep is at best a flaky solution :/

Reply:
yes or we can push a solution and find a better one later :) 
since it seems that there is not so much volunteer currently we can revert if you want but it seems to me that it's better to have something which works even if it not super clean

Reply:
My debugging mind feels like I'd actually like to have people with problems so we can find the source. But I agree that I didn't expect you to have any, I consistently get everything up & running from the get go :sweat_smile:

Feel free to open a PR after that to fix that bug (I imagine you have it on the other commands as well then?)